src/CodeGen_LLVM.cpp

```diff
@@ -2231,6 +2231,25 @@ void CodeGen_LLVM::visit(const Call *op) {
     } else if (op->is_intrinsic(Call::interleave_vectors)) {
         internal_assert(0 < op->args.size());
         value = interleave_vectors(op->type, op->args);
+    } else if (op->is_intrinsic(Call::slice_vector)) {
+        const IntImm *num_vector_args = op->args[0].as<IntImm>();
```

Comment:
Yes, that is the first todo in my comment right up top. I'll implement this now. wanted to simultaneously get your comments on every thing else too. Hence the pull request.